src/json/members/exwang.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Ethan Wang",
+  "pennkey": "exwang",
+  "major": "Cognitive Science and Computer Science",
+  "school": "SAS",
+  "bio": "Cog Sci and CIS major interested in trying new restaurants!",
+  "hometown": "Naperville, IL",
+  "team": "OHQ",
```

Comment:
@ethanwang-04 can we change this to match the other office hour queue team's `team` field? Lemme know once this is resolved and I'll approve thanks!